content/news/buzz/june-2024-status-update/contents.lr

```diff
@@ -0,0 +1,63 @@
+title: June 2024 Status update
+---
+author: Russell Keith-Magee
+---
+body:
+
+In June, BeeWare made a lot of progress, landing lots of new features - many of which have been the subject of requests and discussions for a long time.
+
+What we've done
+----------------
+
+* We `added support for packaging console apps to Briefcase <https://github.com/beeware/briefcase/pull/1781>`__. This involved adding a new ``.pkg`` packaging type for macOS, as well as a number of other improvements to the stub app and MSI packaging on Windows. In addition to providing a way to package console apps on macOS, the addition of ``.pkg`` format means it is now possible to distribute apps using centralise device management tooling (such as Kandji or FleetDM).
+
+* We modified Briefcase to `distribute the stub binaries as standalone artefact, rather than including them in the app template <https://github.com/beeware/briefcase/pull/1849>`__. We also `significantly optimised how Briefcase uses templates <https://github.com/beeware/briefcase/pull/1855>`__. Between these two changes, the download size required for a first project on macOS has been reduced from around 200MB, to less than 4MB.
```

Comment:
```suggestion
* We modified Briefcase to `distribute the stub binaries as standalone artefacts, rather than including them in the app template <https://github.com/beeware/briefcase/pull/1849>`__. We also `significantly optimised how Briefcase uses templates <https://github.com/beeware/briefcase/pull/1855>`__. Between these two changes, the download size required for a first project on macOS has been reduced from around 200MB, to less than 4MB.
```